src/api/app/components/bs_request_history_element_component.rb

```diff
@@ -10,4 +10,28 @@ def initialize(element:)
 
     @element = element
   end
+
+  private
+
+  def icon
+    case @element.type.demodulize
+    when 'ReviewAccepted', 'RequestAccepted'
+      tag.i(nil, class: 'fas fa-check text-success')
+    when 'ReviewDeclined', 'RequestDeclined'
+      tag.i(nil, class: 'fas fa-times text-danger')
+    when 'RequestReviewAdded'
+      tag.i(nil, class: 'fas fa-2xs fa-circle text-warning')
+    else
+      tag.i(nil, class: 'fas fa-code-commit text-dark')
+    end
+  end
+
+  # While all history elements possibly have a comment, not all of them are from an actual human...
+  def element_with_comment_from_human?
```

Comment:
For an example of a history element with a comment which isn't from an actual human, see this:

https://github.com/openSUSE/open-build-service/blob/5d51c3548328a27e64991ccba6f3ba1873a3e218/src/api/app/models/bs_request.rb#L454-L455